dpsg/docs/getting-started/connect.md

```diff
@@ -41,12 +46,22 @@ Some payment providers don't like passing `http://localhost` as a redirect URL (
 
 ### Config Explained
 
-(config : type : default : description)
+(config : type : default value : description)
 
 - `redirectBaseUrl` : string : "" : The base url to be redirected to after payments (usually your client app)
 - `redirectUrl` : string : "/checkout/pending" : The url to be redirected to after payments (usually your client app)
 - `serviceUser` : string : "" : The DPSG user name, in the following format "org:project:env" e.g. "deity:falcon:production"
 - `serviceToken` : string : "" : The DPSG token associated with the user above
+- `region` : string : null : The region of the DPSG instance you wish to use. This defaults to `null` which connects to our European instance. Accepted values are `eu` (Europe), `us` (America) and `au` (Australia).
+
+** Local Config **
+
+These config values are only needed when you're running your DEITY app locally.
+
+(config : type : default value : description)
+
+- `messageQueue.host` : string: "" : This is the host for the message queue. It should be 1 of 3 URLs depending on the region your app is running in: `rabbit.deity.cloud` (Europe), `rabbit.us.deity.cloud` (America), `rabbit.au.deity.cloud` (Australia)
+- `messageQueue.protocol` : string: `"amqp"` : The protocol used by our message queue. When used locally - this needs to be set to `amqps`.
```

Comment:
@hobadams no, setting a region will be enough for the module to handle the right AMQP host. 

`region` controls the REST API **and** AMQP hosts